MusicNotationKit/MusicNotationKit/NoteCollection.swift

```diff
@@ -24,6 +24,8 @@ public protocol NoteCollection {
      `NoteCollection`. Combine this with `noteDuration`.
      */
     var noteTimingCount: Int { get }
+	
+	func note(at index: Int) throws -> Note 
```

Comment:
Make sure you set your Xcode indentation properly.

Go to preferences -> text editing and check the box "Include whitespace-only lines". Go to the indentation tab and set it to Spaces and 4.